### PR TITLE
[OYPD-495] Adding default green focus to links

### PR DESCRIPTION
**Pull request by david-hernandez**
Check focus of links. This won't change every link, because some places have :focus defined, but this will set the default and will work on the top nav and links like the footer menu.

screen shot 2017-06-19 at 4 14 32 pm
screen shot 2017-06-19 at 4 14 25 pm

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1648 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1648 |
| Installation process | http://install.openy-dev.ffwua.com/build1648/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1648/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1648/profiles/contrib/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1655 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1655 |
| Installation process | http://install.openy-dev.ffwua.com/build1655/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1655/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1655/profiles/contrib/openy/build/reports/behat

File: themes/openy_themes/openy_rose/scss/state/_state.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+a:focus,
+.dropdown-toggle:focus {
+  outline: 5px auto #69ff00;
```
Comment (alexschedrov) [inline themes/openy_themes/openy_rose/scss/state/_state.scss]: Let's move this color to variable.

**Comment by alexschedrov** [comment]
1. Focus for gallery arrows should be green as well
image

2. Buttons focus should be green
image

image

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by AlexNetman** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1662 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1662 |
| Installation process | http://install.openy-dev.ffwua.com/build1662/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1662/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build1662

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1662/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1663 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1663 |
| Installation process | http://install.openy-dev.ffwua.com/build1663/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1663/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build1663

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1663/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1665 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1665 |
| Installation process | http://install.openy-dev.ffwua.com/build1665/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1665/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build1665

**Comment by AlexNetman** [comment]
image
In the latest PR shadowbox was added around light-green outline. It looks good allow us to say that we are having contrast 3:1 for all possible combinations:
image

But that brings severe visual issues. When I click on blog title using mouse button:
I see grey shadowbox for a while
image
And when I click on Logo, I see grey box over it:
image

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1665/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1667 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1667 |
| Installation process | http://install.openy-dev.ffwua.com/build1667/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1667/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build1667

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1667/profiles/contrib/openy/build/reports/behat

**Review by AlexNetman — APPROVED** [review]

**Review by alexschedrov — APPROVED** [review]

**Pull request merged**